src/models/raffle.js

```diff
@@ -79,7 +80,8 @@ module.exports.listEntries = function(from, to, localpart) {
     'SELECT smtp_from as from, created as received, subject, rfc822 ' +
     'FROM request_dump.relay_messages ' +
     toWhereStr('WHERE', where) +
-    'ORDER BY created ASC',
+    'ORDER BY ' + orderBy +
```

Comment:
you need to use prepared queries for this (or sanitize the orderby and limit inputs): since you're taking the what is passed from the request and creating a SQL statement from it, you're open to a SQL injection attack.